### PR TITLE
feat:added snackbar

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/155018586-2758fe11-14b4-4a58-a790-45100e8ba922.mp4

### CODE REVIEW

File: components/snackbar/snackbar.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const snackbarContainer = document.querySelector(".snackbar-containerlive");
+const SnackBtn = document.querySelector(".snacbar-btn");
+snackbarContainer.style.display = "none"
+SnackBtn.addEventListener("click",()=>{
+    console.log("sk");
```
Comment (shuklahimanshu81) [inline components/snackbar/snackbar.js:5]: Remove console statement from code.

**Review by shuklahimanshu81 — APPROVED** [review]
Everything else is fine. Good to merge.

**Review by abhij1607 — APPROVED** [review]
Well-structured, clean code, good use variables
Fine to Merge

**Review by aryan-13 — APPROVED** [review]
Code looks good to merge

**Review by ZuberDunge — APPROVED** [review]

**Pull request merged**